### DISCUSSION TITLE
MET Norway Nordic vs their public locationforecast API

### DISCUSSION THREAD

**Discussion by drmrbrewer** [open-meteo/open-meteo#178] [Q&A]
For *Nordic locations*, how does the [MET Norway API](https://open-meteo.com/en/docs/metno-api) from open-meteo differ from met.no's public [locationforecast API](https://api.met.no/weatherapi/locationforecast/2.0/documentation).  From [this page](https://docs.api.met.no/doc/locationforecast/datamodel) it seems like `locationforecast` also uses MetCoOp Ensemble Prediction System (MEPS), so the data would be expected to be the same?  As explained on the page, `locationforecast` also expands coverage beyond the Nordic region with other models.

**Comment by patrick-zippenfenig** [comment]
The used `MET Nordic` dataset is based on `MetCoOp MEPS`, downscaled to 1km with additional nowcasting and hourly updates. I am actually surprised that their API is not using this as a default.

Afterwards they appear to offer 9 km ECMWF. This would actually be against the licence agreement from ECMWF. Maybe as a member state they are allowed to do it.

**Reply by drmrbrewer** [reply]
Thanks for the reply!  So, downscaling to 1 km in the `MET Nordic` dataset is probably the main difference?  Because their "Nordic forecasts are updated once every hour".

On the flipside, this is apparently a feature of data from their API:

> We also use extensive post-processing to correct for errors in the weather model output caused by unresolved features such as cold pools, inversions, urban heat islands, and an intricate coastline. This is one of the reasons why our forecasts may differ from other forecast providers using MEPS.

Regarding spatial res of ECMWF, they refer to a spatial res of 18 km for the Nordic and Arctic regions but 9 km for "rest of world".  Bit odd that they would use lower resolution data around Norway, or maybe the article is wrong about this.

**Reply by patrick-zippenfenig** [reply]
The `MET Nordic` is doing such post-processing. In this case, it is likely the same data

> For medium range forecasts (2 - 10 days) the 51 member ensemble forecast from [ECMWF](https://www.ecmwf.int/en/forecasts/datasets/set-iii) is used. It it updated twice pr day. Horizontal resolution is approximately 18 km.

Arctic seems to use 2.5 km as well. 18 km is probably referring to the ECMWF ensemble forecast with 51 members.
